src/main/resources/schemas/nhinc/common/AuditQueryLog.xsd

```diff
@@ -0,0 +1,69 @@
+            <xsd:element name="EventOutcomeIndicator" minOccurs="0" type="xsd:integer" />
+            <xsd:element name="eventBeginDateTime" type="xsd:dateTime" />
+            <xsd:element name="eventEndDateTime" minOccurs="0" type="xsd:dateTime" />
+            <xsd:element name="RemoteHcid" minOccurs="0" type="xsd:string" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="QueryAuditEventsRequestType" type="tns:QueryAuditEventsRequestType"/>
+    <xsd:complexType name="QueryAuditEventsResponseType" >
+        <xsd:sequence>
+            <xsd:element name="UserId" minOccurs="0" type="xsd:string" />
+            <xsd:element name="EventType"  type="xsd:string" />
+            <xsd:element name="EventId"  type="xsd:string" />
+            <xsd:element name="EventOutcomeIndicator"  type="xsd:integer" />
+            <xsd:element name="EventTimestamp" type="xsd:dateTime" />
+            <xsd:element name="RemoteHcid" minOccurs="0" type="xsd:string" />
+            <xsd:element name="RelatesTo" minOccurs="0" type="xsd:string" />
+            <xsd:element name="Direction"  type="xsd:string" />
+            <xsd:element name="Id"  type="xsd:long" />
+            <xsd:element name="RequestMessageId"  minOccurs="0" type="xsd:string" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="QueryAuditEventsResponseType" type="tns:QueryAuditEventsResponseType"/>
+    <xsd:complexType name="QueryAuditEventsRequestByRequestMessageId">
+        <xsd:sequence>
+            <xsd:element name="RequestMessageId"  minOccurs="0" type="xsd:string" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="QueryAuditEventsRequestByRequestMessageId" type="tns:QueryAuditEventsRequestByRequestMessageId"/>
+    
+    <xsd:complexType name="QueryAuditEventsResponseByRequestMessageId" >
+        <xsd:sequence>
+            <xsd:element name="UserId" minOccurs="0" type="xsd:string" />
+            <xsd:element name="EventType"  type="xsd:string" />
+            <xsd:element name="EventId"  type="xsd:string" />
+            <xsd:element name="EventOutcomeIndicator"  type="xsd:integer" />
+            <xsd:element name="EventTimestamp" type="xsd:dateTime" />
+            <xsd:element name="RemoteHcid" minOccurs="0" type="xsd:string" />
+            <xsd:element name="RelatesTo" minOccurs="0" type="xsd:string" />
+            <xsd:element name="Direction"  type="xsd:string" />
+            <xsd:element name="Id"  type="xsd:long" />
+            <xsd:element name="RequestMessageId"  minOccurs="0" type="xsd:string" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="QueryAuditEventsResponseByRequestMessageId" type="tns:QueryAuditEventsResponseByRequestMessageId"/>
+    
+    <xsd:complexType name="QueryAuditEventsBlobRequest" >
+        <xsd:sequence>
+            <xsd:element name="Id" type="xsd:long" />
+            <xsd:element ref="adtmsg:AuditMessage" />
+        </xsd:sequence>
```

Comment:
Not sure why we would need a adtmsg:AuditMessage element in QueryAuditEventsBlobRequest. The Id element is enough to query blob.